erpnext/patches/v12_0/move_item_tax_to_item_tax_template.py

```diff
@@ -0,0 +1,80 @@
+import frappe
+import json
+from six import iteritems
+
+old_item_taxes = {}
+item_tax_templates = {}
+rename_template_to_untitled = []
```

Comment:
Avoid global variables in python

https://stackoverflow.com/questions/19158339/why-are-global-variables-evil